patchwork/urls.py

```diff
@@ -36,6 +36,11 @@
         patch_views.patch_list,
         name='patch-list',
     ),
+    path(
+        'project/<project_id>/series-list/',
```

Comment:
We should align this with the URL structure of the rest of the URLs. For example, `project/<project_id>/series/`.

We'll also eventually need a series detail view (and a new-style mbox and patch view), but it's okay to tackle that separately.